tests/OK/292.lp

```diff
@@ -0,0 +1,21 @@
+set flag "print_domains" on
+
+symbol R : TYPE
+constant symbol zero : R
+symbol d2_1 : (R ⇒ R ⇒ R) ⇒ (R ⇒ R ⇒ R)
+
+// The following are semantically equivalent
+rule d2_1 (λ_ y, &F[y]) → λ_ _,zero // warning
+rule d2_1 (λ_ y, &_[y]) → λ_ _,zero // warning
+rule d2_1 (λ_ y, _    ) → λ_ _,zero // no warning
+
+// The following two are NOT semantically the same
+rule d2_1 (λx y, &_[y]) → λ_ _,zero // warning
```

Comment:
Why would you get a warning here?

Reply:
I think it is just a copy paste error, there is no warning.